src/App.js

```diff
@@ -1,12 +1,14 @@
 import { Gallery, Rules } from "./views";
 import { Navbar } from "./components";
+import TeamCard from "./components/TeamCard/TeamCard";
```

Comment:
Don’t import directly. Import it through component index. It’s easier to manage look nice.